content/infrastructure/rippled/installation/build-on-linux-mac-windows.md

```diff
@@ -0,0 +1,19 @@
+---
+html: build-on-linux-mac-windows.md
+parent: install-rippled.html
+blurb: Build rippled on Linux, Mac (macOS), or Windows
+labels:
+  - Core Server
+  - Blockchain
+top_nav_grouping: Popular Pages
+---
+# Build `rippled` on Linux, Mac, or Windows
+
+Building `rippled` across various platforms requires a C++ development environment. You will need tools like Git, Python, Conan, CMake, and a suitable C++ compiler.
```

Comment:
```suggestion
Building `rippled` across various platforms such as Windows, Linux, or macOS requires a C++ development environment. You will need tools like Git, Python, Conan, CMake, and a suitable C++ compiler.
```

Take this opportunity to show that the order the platforms are listed in doesn't matter, and Mac == macOS